crates/language/src/language_settings.rs

```diff
@@ -119,6 +120,8 @@ pub struct AllLanguageSettingsContent {
     /// The settings for individual languages.
     #[serde(default, alias = "language_overrides")]
     pub languages: HashMap<Arc<str>, LanguageSettingsContent>,
+    #[serde(default)]
+    pub associations: HashMap<Arc<str>, Vec<Arc<str>>>
```

Comment:
Curious, do we really need `Arc<str>` all over the place or could use just a `String`?

Reply:
I use `Arc<str>` just to maintain consistency with the line above ` pub languages: HashMap<Arc<str>, LanguageSettingsContent>`, and I also agree with you to use just a `String`.